configs/components/cfpropertylist.rb

```diff
@@ -1,7 +1,7 @@
 component "cfpropertylist" do |pkg, settings, platform|
   pkg.version "2.2.7"
   pkg.md5sum "be2fc619084422cc28cb68f6084bea91"
-  pkg.url "http://buildsources.delivery.puppetlabs.net/CFPropertyList-2.2.7.tar.gz"
+  pkg.url "https://rubygems.org/downloads/CFPropertyList-#{pkg.get_version}.gem"
```

Comment:
Ah, when we snatched the source tarball it probably came from a Debian or RHEL package back in the day and we preserved the case over time because :skull::skull::skull:.